shared/actions/chat.js

```diff
@@ -2086,7 +2086,7 @@ function * _getInboxAndUnbox ({payload: {conversationIDKey}}: Constants.GetInbox
   const param: ChatTypes.localGetInboxAndUnboxLocalRpcParam = {
     identifyBehavior: TlfKeysTLFIdentifyBehavior.chatGui,
     query: {
-      convID: keyToConversationID(conversationIDKey),
+      convIDs: [keyToConversationID(conversationIDKey)],
```

Comment:
This is fine , we actually call this function multiple times since it only used to take in one id, so now we can collapse all those calls. I'll ticket that